e2e/cypress/integration/messaging/message_spec.js

```diff
@@ -182,4 +182,27 @@ describe('Message', () => {
         // * Focus to remain in the RHS text box
         cy.get('#reply_textbox').should('be.focused');
     });
+
+    it('MM-T336 Image thumbnail - expanded RHS', () => {
```

Comment:
This test belongs to "Files and Attachments" functional area. So, please create a new `cypress/integration/files_and_attachments` folder and move it into a new spec file. (ref: item 2 on [Folder and File Structures](https://developers.mattermost.com/contribute/webapp/end-to-end-tests/#folder-and-file-structures) 😄 )